app-tracking-protection/vpn-impl/src/main/java/com/duckduckgo/mobile/android/vpn/heartbeat/VpnServiceHeartbeat.kt

```diff
@@ -74,6 +75,7 @@ class VpnServiceHeartbeat @Inject constructor(
         Timber.v("onVpnStopped called")
         when (vpnStopReason) {
             ERROR -> Timber.v("HB monitor: sudden vpn stopped $vpnStopReason")
+            RESTART -> Timber.v("HB monitor: restarting vpn $vpnStopReason")
```

Comment:
I don't think we need to do anything here when we do a restart, let me know if it's not the case.